filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -2927,6 +2931,25 @@ void OpenGLDriver::executeEveryNowAndThenOps() noexcept {
     }
 }
 
+void OpenGLDriver::runAtNextRenderPass(void* token, std::function<void()> fn) noexcept {
+    assert_invariant(mRunAtNextRenderPassOps.find(token) == mRunAtNextRenderPassOps.end());
+    mRunAtNextRenderPassOps[token] = std::move(fn);
+}
+
+void OpenGLDriver::cancelRunAtNextPassOp(void* token) noexcept {
+    mRunAtNextRenderPassOps.erase(token);
+}
+
+void OpenGLDriver::executeRenderPassOps() noexcept {
+    auto& ops = mRunAtNextRenderPassOps;
+    if (!ops.empty()) {
```

Comment:
UTILS_UNLIKELY maybe?

Reply:
I was going to use it and then I thought executeRenderPassOps() is actually generic, so we can't really tell if it's likely or not.